PR for SRVKE-1386: [DOC] Create listed 2 times in Knative Eventing installation (#62288)
openshift/openshift-docs: modules/serverless-install-eventing-web-console.adoc

. Click *Create Knative Eventing*.

> . In the *Create Knative Eventing* page, you can choose to configure the `KnativeEventing` object by using either the default form provided, or by editing the YAML file.

Reviewer: ```suggestion
. In the *Create Knative Eventing* page, you can configure the `KnativeEventing` object by using either the form provided, or by editing the YAML file.
```
suggestion for minimalism

* Use the form for simpler configurations that do not require full control of `KnativeEventing` object creation.

. Click *Create*.
+

> * To have full control of `KnativeEventing` object creation, you can edit the YAML file for more complex configurations. You can access the YAML file by clicking the *edit YAML* link in the top right of the *Create Knative Eventing* page.

Reviewer: ```suggestion
* Edit the YAML file for more complex configurations that require full control of `KnativeEventing` object creation. To access the YAML editor, click *edit YAML* on the *Create Knative Eventing* page.
```
some comments:
- "Edit the..." is more in line with the previous bullet language that says "Use the..." rather than "You can..."
- Wording sounded a bit strange in the first sentence to me, have added suggestions to clarify.
- per IBM style guide, do not use "top right, top left" etc for accessibility reasons: "Accessibility and globalization requirement: People who have limited vision, or who use interfaces that are translated into a bidirectional language, might not understand information if it is conveyed only by location. Provide additional descriptive information. For example, write The tables are displayed in the Table List pane, which is in the upper-right corner of the window, not The list of tables is displayed in the upper-right corner of the window."

> * Using the form is recommended for simpler configurations that do not require full control of `KnativeEventing` object creation.

Reviewer 2: ```suggestion
* Use the form for simpler configurations that do not require full control of `KnativeEventing` object creation.
```

+

> * Optional: If you are configuring the `KnativeEventing` object using the form, make any changes that you want to implement for your Knative Eventing deployment.

Reviewer 2: ```suggestion
* Optional: For Knative Eventing deployment, configure the `KnativeEventing` object by using the form.
```

Author: Done! Thank you! :)

> * Optional: If you are configuring the `KnativeEventing` object using the form, make any changes that you want to implement for your Knative Eventing deployment.

Author: Done! Thank you! :)

. Click *Create*.
+

> * Editing the YAML is recommended for more complex configurations that require full control of `KnativeEventing` object creation. You can access the YAML by clicking the *edit YAML* link in the top right of the *Create Knative Eventing* page.

Reviewer 2: ```suggestion
* To have full control of `KnativeEventing` object creation, you can edit the YAML file for more complex configurations. You can access the YAML by clicking the *edit YAML* link in the top right of the *Create Knative Eventing* page.

```

Author: Done! Thank you! :)

+

> * Optional: If you are configuring the `KnativeEventing` object by editing the YAML, make any changes to the YAML that you want to implement for your Knative Eventing deployment.

Reviewer 2: ```suggestion
* Optional: For Knative Eventing deployment, configure the KnativeEventing object by editing the YAML file.
```

Author: Done! Thank you! :)
